jsonnet/kube-prometheus/components/node-exporter.libsonnet

```diff
@@ -21,6 +21,7 @@ local defaults = {
   },
   listenAddress:: '127.0.0.1',
   filesystemMountPointsExclude:: '^/(dev|proc|sys|run/k3s/containerd/.+|var/lib/docker/.+|var/lib/kubelet/pods/.+)($|/)',
+  exportersExclude:: ['wifi', 'btrfs', 'hwmon'],
```

Comment:
IIUC by default we should not exclude these, you can have the exclude variable and while importing this jsonnet you should define the variable to include collectors to exclude so that default setting doesn't change

Reply:
They were already excluded, I just exposed them as a variable.